g3doc/obtaining-training-data.md

```diff
@@ -0,0 +1,50 @@
+# Obtaining Training Data
+
+This document gives an overview of how to use the Gematria tooling to
+parse/process training data from multiple sources.
+
+## Parsing training data from the BHive dataset
+
+The publically available [BHive](https://github.com/ithemal/bhive) dataset
+has about 300k basic blocks along with throughput data for Ivy Bridge, Haswell,
+and Skylake in a CSV format. Gematria provides native tooling to process these
+CSV files into `.tfrecord` files that can be used with Gematria models.
+
+Every line in the BHive CSV files has the format
+`<machine code of BB in hex>,<throughput in cycles>>`. This is referred to as
+the BHive format. The dataset contains one CSV file per mircoarchitecture.
+To process BHive throughput data for the Intel Skylake
+microarchitecture, we can begin by downloading one of the BHive CSVs:
```

Comment:
I'd even do this as
To process BHive throughput data for the Intel Skylake microarchitecture:

1. download the corresponding BHive CSV:
   ```
   ...
   ```
2. Convert the downloaded CSV file into a `.tfrecord` file:
   ```
   ...
   ```